react/finger-exercises/redux/01 - actions and reducers/books/actions.js

```diff
@@ -18,10 +19,22 @@ The purpose of this object is to have all the action-creating logic in a single
 place. All actions types must only be used inside an actionCreator.
 */
 export default {
+  setFavoriteBook: title => ({
```

Comment:
Notice if you variable is called `title` you do this
```
payload: {
      favoriteBook: title
    }
```
If you variable is called favoriteBook, you do this
```
payload: {
      favoriteBook
    }
```